### PR TITLE
Added command for installing K8s dev tools.

### LINKED ISSUE #308: Add `ujust k8s-dev-tools`
`ujust k8s-dev-tools` should install the bundled kubernetes [file](https://github.com/ublue-os/bluefin/commit/06e3464cf51f10ecaffa924b4d50e528bb29d7f8).

### PR DESCRIPTION
**Pull request by dnikoloski**
## Thank you for contributing to the Universal Blue project!

Please [read the Contributor's Guide](https://universal-blue.org/CONTRIBUTING/) before submitting a pull request.

Closes #308

### CODE REVIEW

**Comment by castrojo** [comment]
Would you mind adding the Kubernetes.Brewfile here too? Otherwise if you run that command on Bazzite it'd fail. 

Plus we'd avoid having bluefin and bazzite diverge, k8s nerds can be happy on both, thanks!

**Comment by dnikoloski** [comment]
> Would you mind adding the Kubernetes.Brewfile here too? Otherwise if you run that command on Bazzite it'd fail.
> 
> Plus we'd avoid having bluefin and bazzite diverge, k8s nerds can be happy on both, thanks!

Where exactly, it seems like you forgot to paste the link?

**Comment by castrojo** [comment]
https://github.com/ublue-os/bluefin/blob/main/system_files/shared/usr/share/ublue-os/homebrew/kubernetes.Brewfile

**Review by castrojo — APPROVED** [review]

**Comment by castrojo** [comment]
I can do a follow up on this one with the bluefin/bazzite-cli ones so we can centralize.

**Review by HikariKnight — APPROVED** [review]

**Comment by HikariKnight** [comment]
ideally i would have liked it to use the `install-` verb, however it will get too wordy and it is doing something specific that will be fine without a verb.
